### ISSUE TITLE
react-menu-footer-classic-modern

### ISSUE THREAD

**Issue by jurivero** [pnp/sp-dev-fx-extensions#237]
Hi,

I have another test that have also issues when running gulp please see below:
C:\Projects\sp-dev-fx-extensions\samples\react-menu-footer-classic-modern\SPFx>gulp test
gulp test[21384]: c:\ws\src\node_contextify.cc:626: Assertion `args[1]->IsString()' failed.
 1: 00007FF7E81EF04A v8::internal::GCIdleTimeHandler::GCIdleTimeHandler+5114
 2: 00007FF7E81CA0C6 node::MakeCallback+4518
 3: 00007FF7E81CA17F node::MakeCallback+4703
 4: 00007FF7E819F630 node::DecodeWrite+13024
 5: 00007FF7E89CC122 v8::internal::OptimizingCompileDispatcher::Unblock+59890
 6: 00007FF7E89CD289 v8::internal::OptimizingCompileDispatcher::Unblock+64345
 7: 00007FF7E89CC5CC v8::internal::OptimizingCompileDispatcher::Unblock+61084
 8: 00007FF7E89CC4EB v8::internal::OptimizingCompileDispatcher::Unblock+60859
 9: 0000002039BDC5C1

C:\Projects\sp-dev-fx-extensions\samples\react-menu-footer-classic-modern\SPFx>gulp serve --nobrowser
gulp serve --nobrowser[22868]: c:\ws\src\node_contextify.cc:626: Assertion `args[1]->IsString()' failed.
 1: 00007FF7E81EF04A v8::internal::GCIdleTimeHandler::GCIdleTimeHandler+5114
 2: 00007FF7E81CA0C6 node::MakeCallback+4518
 3: 00007FF7E81CA17F node::MakeCallback+4703
 4: 00007FF7E819F630 node::DecodeWrite+13024
 5: 00007FF7E89CC122 v8::internal::OptimizingCompileDispatcher::Unblock+59890
 6: 00007FF7E89CD289 v8::internal::OptimizingCompileDispatcher::Unblock+64345
 7: 00007FF7E89CC5CC v8::internal::OptimizingCompileDispatcher::Unblock+61084
 8: 00007FF7E89CC4EB v8::internal::OptimizingCompileDispatcher::Unblock+60859
 9: 00000180E895C5C1

**Comment by hugoabernier** [comment]
Closing this issue as incomplete.

If you're still experiencing this issue, feel free to open a new issue. If you do, please take the time to fill the issue template provided so that we can get you the help you deserve.
